simpledotorg/simple-server
# Convert chart.js import statements to single npm package import

**Story card:** [sc-10491](https://app.shortcut.com/simpledotorg/story/10491/change-chartjs-to-npm-import?vc_group_by=week&ct_workflow=all&cf_workflow=500000031)

## Because

ChartJS is currently loaded by calling the script in individual modules (8 in total).
This changes it to an NPM package import thats imported to the app at application.js

## This addresses

- add chart.js npm package
- import globally
- load chartjs into app
- remove all import statements
- 
## Test instructions

Switch to the branch
run 'npm install'
ensure that charts load as expected on reports page (HTN and DM) and on the trends page
